DfE.FindInformationAcademiesTrusts/Pages/Trusts/_SourceList.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@model ITrustsAreaModel
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      <span>Source and updates</span>
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    @foreach (var source in Model.DataSources)
+    {
+      <dl role="list">
```

Comment:
I think this needs to be outside the for loop?

We can look at the lists two ways:

How you have it above, with one dl for the whole element:
- Each item has a descriptive title which is the data source
- each item has two descriptive elements which give info about that source 

OR 

an unordered list containing descriptive lists which would be:

- each `<li>` in `<ul>` contains a `<dl>`
- the dl remains inside the loop
- for each line above, the `<dt>` is the bit before the colon, the `<dd>` is the bit after

Also does a dl need a role? I think screen readers all read them out differently. It doesn't have a role elsewhere in our app, and [GOV.UK summary list](https://design-system.service.gov.uk/components/summary-list/) doesn't